offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonView.groovy

```diff
@@ -411,4 +409,13 @@ class CreatePersonView extends VerticalLayout {
         createPersonViewModel.outdatedPerson = null
 
     }
+
+    protected Optional<Organisation> getOrganisation(Affiliation affiliation) {
+        Optional<Organisation> foundOrganisation = Optional.empty()
+        createPersonViewModel.availableOrganisations.each {
+            if(affiliation in (it as Organisation).affiliations) foundOrganisation = Optional.of((it as Organisation))
```

Comment:
I think you can replace the casting by declaring the type of the it variable in the closure to make it more readable.
```suggestion
        createPersonViewModel.availableOrganisations.each {Organisation organisation ->
            if(affiliation in organisation.affiliations) foundOrganisation = Optional.of(organisation)
```